WinImageBuilder.psm1

```diff
@@ -361,16 +361,26 @@ function Download-CloudbaseInit {
         [Parameter(Mandatory=$true)]
         [string]$resourcesDir,
         [Parameter(Mandatory=$true)]
-        [string]$osArch
+        [string]$osArch,
+        [parameter(Mandatory=$false)]
+        [switch]$BetaRelease=$true
+
     )
     Write-Host "Downloading Cloudbase-Init..."
 
-    if ($osArch -eq "AMD64") {
-        $CloudbaseInitMsi = "CloudbaseInitSetup_Stable_x64.msi"
+   if ($BetaRelease) {

```

Comment:
do like this

$msiBuildArchMap = {
    "amd64" = "x64"
    "i386" = "x86"
}
$msiBuildSuffix = ""
if (-not $BetaRelease) {
    $msiBuildSuffix = "_Stable"
}
$CloudbaseInitMsi = "CloudbaseInitSetup{0}_{1}.msi" -f @($msiBuildSuffix, $msiBuildArchMap[$osArch])

And ALWAYS! put spaces before and after operators